apps/src/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundBoard.js

```diff
@@ -48,6 +49,14 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
 
     /** @private {SerialPort} serial port controller */
     this.serialPort_ = null;
+    firehoseClient.putRecord(
```

Comment:
could we pull these firehose events into a helper function? i think the only thing that's different between each event is the `event` property -- everything else is the same. i think it will simplify this code and make these events easier to remove in the future. i'm not very familiar with this area of the code, but there may be a utils file we put them in? or this file is fine

Reply:
great idea! I'll update

Reply:
Decided to just create a function within the same class since it should be removed eventually and was pretty specific to this one use case. https://github.com/code-dot-org/code-dot-org/pull/43732/commits/a4a9341fab36c96fbd919f9a3f46e35962509fef